stl/inc/cvt/jis_0208

```diff
@@ -41,7 +41,7 @@ namespace stdext {
             virtual ~codecvt_jis_0208() noexcept {}

```

Comment:
Some destructors are just `virtual`, but can be just `override`. Not sure if it needs to be changed though.

Reply:
A couple didn't have either.